test/SILOptimizer/utf8_decoding_fastpath.swift

```diff
@@ -1,5 +1,7 @@
 // RUN: %target-swift-frontend -O -emit-sil %s | %FileCheck %s
-// REQUIRES: optimized_stdlib,CPU=x86_64
+// REQUIRES: swift_stdlib_no_asserts
```

Comment:
This simply makes every invocation of the test be in an unsupported configuration, so it's not a reasonable fix.